Cargo.toml

```diff
@@ -43,6 +45,8 @@ rust-crypto = { version = "^0.2", optional = true }
 hacl-star = { version = "=0.0.11", optional = true }
 rand = { version = "0.5.0-pre.2", optional = true }
 ring = { version = "0.13.0-alpha5", optional = true }
+#sodiumoxide = { version = "^0.1.0", optional = true } # We have to use concrete revision until version bump.
+sodiumoxide = { git = "https://github.com/sodiumoxide/sodiumoxide", rev = "d01bbdbd91313453ae244d56973545d9b695af20", optional = true }
```

Comment:
Like you mentioned earlier, we're going to have to wait to merge this until there's a released version of `sodiumoxide` since I can't in turn release `snow` if it depends on any git-sourced crates. Hopefully it doesn't take them too long :).

Reply:
Yes, sure. This PR is not "to-be-merged-asap", but to discuss its state and maybe polish it.